src/js/report_form.js

```diff
@@ -12,24 +13,32 @@
     scheduleCustomInput.closest("tr") ||
     scheduleCustomInput.closest(".form-group");
 
+  if (isRBAC === true){
```

Comment:
`rbac` determines whether default timezone can be changed?  I thought it was Airflow version.

Would a better way be to check if the default timezone is NOT UTC?

Reply:
we only implemented this for rbac because non-rbac has no concept of local time. so we're trying to ensure that our additional fields and time logic isn't add in non-rbac where there's no 'selected-timezone' or 'manual timezone' in the local storage (it would error and the ui would be messed up).

non rbac can still have a default timezone that is not UTC as well as rbac. rbac just has a 2nd timezone it tracks whereas nonrbac only tracks the default in the cfg file

Reply:
So, say people are using a current version of Airflow but have rbac turned off.  Does that mean they can't set a local time?